hugo/layouts/shortcodes/youtube.html

```diff
@@ -0,0 +1,33 @@
+{{- $id := .Get "id" | default (.Get 0) -}}
+{{- $title := .Get "title" | default "" }}
+{{- $image := .Get "image" | default "" }}
+{{- $url := "https://www.youtube.com/watch?v={{ $id }}" }}
+{{- $contentClass := "video__content" }}
+{{- $lazy := cond (or (eq (.Get "lazy") false) (eq (.Get "lazy") "false")) false true -}}
+
+{{ if not $title }}
+    {{- $contentClass = print $contentClass " video__content--no-title" -}}
+{{ end }}
+
+<div class="video" data-youtube="{{ $id }}">
+    {{ if $image }}
+        <img class="video__image" src="{{ $image }}" alt="{{ $title }}"{{ if $lazy }} loading="lazy"{{ end }}>
+    {{ else }}
+        <img class="video__image" src="https://img.youtube.com/vi/{{ $id }}/maxresdefault.jpg" alt="{{ $title }}"{{ if $lazy }} loading="lazy"{{ end }}>
```

Comment:
we're not allowed to load images from youtube because they set marketing cookies so we should remove the fallback and make image required